src/sentry/static/sentry/app/components/pluginConfig.jsx

```diff
@@ -71,6 +71,7 @@ class PluginConfig extends React.Component {
   };
 
   handleTestPlugin = async () => {
+    this.setState({testResults: ''});
```

Comment:
nit: Probably won't ever happen but since `setState` is async, you might want an await to prevent a rare race condition failure.

Reply:
@leeandher Using `await` on `setState` is not really the preferred way of handling race conditions. If you're concerned about that, you should use a callback function as the second argument to `setState`. In this function, I am not relying on the value of `testResults` so there's no race condition that could be a problem.